server/src/main/java/org/candlepin/model/PoolCurator.java

```diff
@@ -682,13 +681,6 @@ public void batchDelete(List<Pool> pools) {
         for (Pool pool : pools) {
             currentSession().delete(pool);
         }
-        for (Pool pool : pools) {
```

Comment:
Given the comment below, what is the risk from not clearing the entitlements here?